backends/graphics/dinguxsdl/dinguxsdl-graphics.cpp

```diff
@@ -122,7 +122,7 @@ void DINGUXSdlGraphicsManager::initSize(uint w, uint h) {
 	if (w > 320 || h > 240) {
 		setGraphicsMode(GFX_HALF);
 		setGraphicsModeIntern();
-		_eventSource->toggleMouseGrab();
```

Comment:
You mean the event recorder? I don't think it ever worked with the Dingux backend, did it? But I can't say I can test Dingux anyway... But I don't see how this should have any effect since the event recorder doesn't do anything in the old SdlEventSource::toggleMouseGrab function. Maybe you can explain this in depth?